frontend_build/src/webpack.config.base.js

```diff
@@ -77,9 +77,9 @@ var config = {
   ],
   resolve: {
     alias: {
-      'kolibri_module': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'kolibri_module'),
-      'core-base': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'core-base'),
-      'core-theme.styl': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'core-theme.styl'),
+      'kolibri_module': path.resolve('kolibri/core/assets/src/kolibri_module'),
+      'core-base': path.resolve('kolibri/core/assets/src/core-base'),
+      'core-theme.styl': path.resolve('kolibri/core/assets/src/core-theme.styl'),
```

Comment:
it's not clear to me from [the Node.js docs](https://nodejs.org/docs/latest/api/path.html#path_path_resolve_path) - do you know if this converts to the correct path separator under Windows?